src/system_metrics/service_mongoose_system_metrics.erl

```diff
@@ -8,6 +8,14 @@
 -define(DEFAULT_REPORT_AFTER, timer:hours(3)).
 -define(TRACKING_ID, "UA-151671255-2").
 -define(TRACKING_ID_CI, "UA-151671255-1").
+-define(MSG_REMOVED_FROM_CONFIG,
+        "Are you sure you don't want to report anything?"
+        "TODO: Explain this better"
+       ).
+-define(MSG_ACCEPT_TERMS_AND_CONDITIONS,
+        "Do you agree with reporting?"
```

Comment:
"In order to improve MongooseIM we are collecting basic system metrics which helps us understand how to developer the product. Take look at {{this}} and {{that}} documentation page(s) to learn more about the metrics we are interested in."